src/libServer/LookupServer.cpp

```diff
@@ -1159,6 +1185,93 @@ Json::Value LookupServer::GetEthBlockTransactionCountByNumber(
   }
 }
 
+Json::Value LookupServer::GetEthTransactionByBlockHashAndIndex(
+    const std::string& inputHash, const std::string& indexStr) const {
+  try {
+    const BlockHash blockHash{inputHash};
+    const auto txBlock = m_mediator.m_txBlockChain.GetBlockByHash(blockHash);
+    const uint64_t index = std::strtoull(indexStr.c_str(), nullptr, 0);
+    return GetEthTransactionFromBlockByIndex(txBlock, index);
+
+  } catch (std::exception& e) {
+    LOG_GENERAL(INFO, "[Error]" << e.what() << " Input: " << inputHash);
+
+    throw JsonRpcException(RPC_MISC_ERROR, "Unable To Process");
+  }
+}
+
+Json::Value LookupServer::GetEthTransactionByBlockNumberAndIndex(
+    const std::string& blockNumberStr, const std::string& indexStr) const {
+  try {
+    TxBlock txBlock;
+    if (blockNumberStr == "latest") {
+      txBlock = m_mediator.m_txBlockChain.GetLastBlock();
+    } else if (blockNumberStr == "earliest") {
+      txBlock = m_mediator.m_txBlockChain.GetBlock(0);
```

Comment:
I think we might need to test on the isolated server + devnet that this shouldn't be `1` - IIRC the isolated server does start at 1, devnet might also depending on how genesis works

Reply:
It's just a matter of interpretation I'd say. 0 is the first valid block mined ever (it's a genesis containing no transactions). So I'd say this is fine to access it here. In addition it's unlikely `earliest` has any useful use-case in our network - I guess no one really uses in on ethereum mainnet either.